lib/package.rb

```diff
@@ -118,28 +119,22 @@ def self.system(*args)
     # 3. The value of `nproc`.strip
     # See lib/const.rb for more details
 
-    # extract command arguments
-    cmd_args = args.select {|arg| arg.is_a?(String) } .join(' ')
-  
-    # extract Kernel.system options (if provided)
-    system_options = if args[-1].is_a?(Hash)
-      { exception: true }.merge(args[-1])
+    # extract env hash
+    if args[0].is_a?(Hash)
+      env = CREW_ENV_OPTIONS_HASH.merge(args[0])
+      args.delete_at(0) # remove env hash from args array
     else
-      { exception: true }
+      env = CREW_ENV_OPTIONS_HASH
```

Comment:
Can you add a check along the lines of `abort unless env.is_a?(Hash)` so we don't break what we're passing to Kernel.system in the future?

Reply:
I think the line 126 already handle it?